src/app/core/navbar/navbar.component.css

```diff
@@ -49,6 +49,12 @@
 
 .navbar-toggler {
   margin-right: 1em;
+  border-color: rgba(255, 255, 255, 0.2);
+}
+
+.navbar-toggler-icon {
+  /* Replace the hamburger icon image with a custom SVG that has custom colours */
+  background-image: url("data:image/svg+xml;charset=utf8,%3Csvg viewBox='0 0 32 32' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath stroke='rgba(255,255,255,0.7)' stroke-width='2' stroke-linecap='round' stroke-miterlimit='10' d='M4 8h24M4 16h24M4 24h24'/%3E%3C/svg%3E");
```

Comment:
Could this svg be saved into `/assets`? It seems to be cluttering up the CSS here.